src/config/configAggregator.ts

```diff
@@ -344,10 +343,10 @@ export class ConfigAggregator extends AsyncOptionalCreatable<JsonMap> {
   }
 
   private resolveProperties(globalConfig: JsonMap, localConfig?: JsonMap): JsonMap {
-    const accumulator: Dictionary<string> = {};
+    const accumulator = envVarsResolve();
```

Comment:
Start wil resolved env vars.

Reply:
Does it make sense to expose a more elaborate abstraction? Something like this:

```typescript
const envVars = new EnvVars();

envVars.get(propertyToEnvName(property.key))
```

And then we could have other useful methods, e.g. `all`, `set`, `unset`, etc...